tests/test_sanity.py

```diff
@@ -29,13 +29,17 @@ def setUpClass(self):
     print('WARNING: This will modify %s, and in theory can break it although it should be restored properly. A backup will be saved in %s_backup' % (EM_CONFIG, EM_CONFIG))
     print()
 
+    os.environ[EMCC_SKIP_SANITY_CHECK_NAME] = '-1'
```

Comment:
-1?

Reply:
I agree, this `-1` looks just like a typo, but in fact, it (and every other string) has opposite meaning to `1`. It would be better to replace it with `os.environ[EMCC_SKIP_SANITY_CHECK_NAME] = 'dontskip'`